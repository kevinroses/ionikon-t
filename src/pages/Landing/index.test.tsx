// Importa React y cualquier otra dependencia necesaria
import React from 'react';

const Landing = () => {
  return (
    <div style={{ background: '#ffffff', /* Otros estilos aquÃ­ */ }}>
     
    </div>
  );
}

export default Landing;
