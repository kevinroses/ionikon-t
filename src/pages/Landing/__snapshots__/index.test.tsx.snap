// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`disable nft on landing page does not render nft information and card 1`] = ``;
